scripts/scil_evaluate_bundles_individual_measures.py

```diff
@@ -94,25 +102,58 @@ def compute_measures(filename_tuple):
     density = compute_tract_counts_map(streamlines, dimensions)
     endpoints_density = get_endpoints_density_map(streamlines, dimensions)
 
-    span_list = list(map(compute_span, list(sft.streamlines)))
+    span_list = list(map(compute_span, streamline_cords))
     span = float(np.average(span_list))
     curl = length_avg / span
     volume = np.count_nonzero(density) * np.product(voxel_size)
     diameter = 2 * np.sqrt(volume / (np.pi * length_avg))
     elon = length_avg / diameter
 
+    roi = np.where(density != 0, 1, density)
+    surf_area = approximate_surface_node(roi) * (voxel_size[0] ** 2)
+    irregularity = surf_area / (np.pi * diameter * length_avg)
+
+    endpoints_map_head, endpoints_map_tail = \
+        get_head_tail_density_maps(sft.streamlines, dimensions)
+    endpoints_map_head_roi = \
+        np.where(endpoints_map_head != 0, 1, endpoints_map_head)
+    endpoints_map_tail_roi = \
+        np.where(endpoints_map_tail != 0, 1, endpoints_map_tail)
+    end_sur_area1 = \
+        approximate_surface_node(endpoints_map_head_roi) * (voxel_size[0] ** 2)
+    end_sur_area2 = \
+        approximate_surface_node(endpoints_map_tail_roi) * (voxel_size[0] ** 2)
+
+    endpoints_coords_head, endpoints_coords_tail = \
+        compute_endpoints_coordinates(sft)
```

Comment:
This function can be removed, since you already computed both endpoints mask earlier, you can do:
`endpoints_coords_head = np.array(np.where(endpoints_map_head_roi)).T`